packages/at_onboarding_flutter/lib/screen/at_onboarding_home_screen.dart

```diff
@@ -574,6 +596,12 @@ class _AtOnboardingHomeScreenState extends State<AtOnboardingHomeScreen> {
       }
 
       _inprogressDialog.close();
+
+      if (authResponse != AtOnboardingResponseStatus.authSuccess ||
+          atSignStatus == ServerStatus.teapot) {
+        _onboardingService.setAtsign = previousAtsign;
+      }
+
```

Comment:
Actual change for this file is here, the rest is formatting from 120 to 80, as this is a published package.